packages/core/README.md

```
@@ -62,6 +89,43 @@ interface FirebaseConfig {
 }
 ```
 
+## JSON Schemas
+
+The package ships JSON Schemas (Draft 2020-12) for validating model files in editors and CI:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)